copier.yaml

```diff
@@ -2,16 +2,21 @@ _subdirectory: ./template_content
 
 _tasks:
   - "echo '==== Checking compatibility with the cli ==='"
-  - '"{{ python_path if python_path else _copier_python }}" pre_init.py'
+  # Execute OS-specific version check script (with self-destruction)
+  - command: |
```

Comment:
As discussed I don't think this is needed anymore, given 2.0.0 was launch 1 year ago.